### DISCUSSION TITLE
403 Forbidden error in accessing the page post-installation

### DISCUSSION THREAD

**Discussion by onPlatinum** [KelvinTegelaar/CIPP#1706] [Q&A]
Hi there,
   I have just installed CIPP by following the installation steps on the CIPP website. The resources are created in Azure without an issue after clicking on the "Deploy to Azure" button, however, in the next step, the "post install configuration", https://docs.cipp.app/setup/installation/addyourself, once I added myself to the invite and logged on, I get a 403 Forbidden error on the browser webpage. Can someone please shed some light on where I have gone wrong and how to fix it?

Thanks
Ji

**Comment by KelvinTegelaar** [comment]
Not the place for support, only sponsors get support.
